### PR TITLE
Added order and check_params_cbs parameter documentation (also to changelog)

### PR DESCRIPTION
**Pull request by MShekow**
Added order parameter to 2.0 changelog. Added check_params_cb parameter documentation to waitSignals. Added check_params_cbs and order parameter documentation to waitSignals.

### CODE REVIEW

File: docs/signals.rst [hunk 1]
```diff
@@ -68,6 +68,31 @@ value of the ``raising`` parameter of the ``qtbot.waitSignal`` and
 Calls which explicitly pass the ``raising`` parameter are not affected.
 
 
+check_params_cb parameter
+---------------------------------
+
+.. versionadded:: 2.0
+
+If the signal has parameters you want to compare with expected values, you can pass 
+``check_params_cb=some_callable(*parameters)`` that compares the provided signal parameters to some expected parameters.
```
Comment (The-Compiler) [inline docs/signals.rst]: The `(*parameters)` seems confusing to me here - why not leave it off?

File: docs/signals.rst [hunk 2]
```diff
@@ -110,6 +135,65 @@ the ``raising`` parameter::
...
         assert_application_results(app)
 
+check_params_cbs parameter
+^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Corresponding to the ``check_params_cb`` parameter of ``waitSignal`` you can use the ``check_params_cbs``
+parameter to check whether one or more of the provided signals are emitted with expected parameters.
+Provide a ``list`` of callables, each matching the signature of the corresponding signal
```
Comment (The-Compiler) [inline docs/signals.rst]: nitpick: I don't think `list` should be monospaced here

**Comment by The-Compiler** [comment]
Great docs! Note that the title underlining should be the same length as the title, otherwise rst-lint will fail. I also added some minor comments.

**Comment by nicoddemus** [comment]
Thanks a lot! :smile:

**Comment by MShekow** [comment]
No problem ;)

**Pull request merged**